database/database.go

```diff
@@ -140,6 +142,36 @@ func (d *Database) GetUser(username string) (user model.User, err error) {
 	return user, nil
 }
 
+// GetUsers will return a list of users whose username matches a search string.
+func (d *Database) GetUsers(search string) ([]model.User, error) {
+	users := []model.User{}
+	sqlStatement := `
+		SELECT Id, Username
+		FROM (SELECT Id, to_tsvector(Username) as lex, UserRole, Username
+				FROM board.user) doc
+		WHERE doc.lex @@ to_tsquery($1) AND UserRole != 2`
```

Comment:
I actually dug it here where you had the UserRole not hard coded, any reason to switch it to having it hard coded?